composer/utils/ddp.py

```diff
@@ -141,6 +141,49 @@ def all_reduce(
                        "The mosaic trainer will automatically do this for you.")
 
 
+def broadcast(tensor: torch.Tensor, src: int = 0) -> None:
```

Comment:
Defaulting `src` to 0 seems a little odd when the Pytorch equivalent has no default value.